lib/galaxy/jobs/actions/post.py

```diff
@@ -228,7 +228,7 @@ def execute(cls, app, sa_session, action, job, replacement_dict):
         for dataset_assoc in job.output_datasets:
             if action.output_name == '' or dataset_assoc.name == action.output_name:
                 for k, v in action.action_arguments.items():
-                    if v != '':
+                    if v and v != '':
```

Comment:
This is equivalent to just:
```python
if v:
```

By looking at the following code it seems that `v` could be a number, so if `v == 0` the following code won't be executed, which seems to be fine to me due to the `if` at line 236, but worth to note.

Reply:
Good point, thanks @nsoranzo